pkg/arvo/app/contact-view.hoon

```diff
@@ -126,18 +133,24 @@
   ?-  -.act
       %create
     ?>  ?=([@ *] path.act)
+    %+  weld
     :~  (group-poke [%bundle path.act])
         (contact-poke [%create path.act])
         (contact-hook-poke [%add-owned path.act])
         (group-hook-poke [%add our.bol path.act])
         (group-poke [%add (~(put in ships.act) our.bol) path.act])
+        (perm-group-hook-poke [%associate path.act [[path.act %white] ~ ~]])
+        (permission-hook-poke [%add-owned path.act path.act])
     ==
+    (create-metadata path.act title.act description.act)
```

Comment:
I don't think you need the `weld` here (and for `%delete` below). You can just `:*` (in place of `:~`) the current list, and add `(create-metadata ...)` as the last item. It's like `:-`ing all these in front of the list that `+create-metadata` produces.

Reply:
I prefer the weld.

Reply:
Then can you please indent their first argument properly? (:

Reply:
ok